package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "islandis-login",
-    "version": "1.1.6",
+    "name": "@island.is/login",
```

Comment:
Maybe call this "@island.is/legacy-login" to distinguish from future login packages?

Reply:
excellent choice